### PR TITLE
Refactored game over screen

### PR DESCRIPTION
**Pull request by 22pilarskil**

### CODE REVIEW

**Comment by 22pilarskil** [comment]
Removed gameoverscreenviewmodel references

**Comment by elizabethahong** [comment]
looks good

**Review by elizabethahong — APPROVED** [review]
looks good, very concise

**Comment by nataliej1217** [comment]
looks good!

**Review by nataliej1217 — APPROVED** [review]
looks good

File: dungeonrunner/app/src/main/java/com/example/dungeonrunner/views/GameOverScreen.java
Comment (etaw2) [inline dungeonrunner/app/src/main/java/com/example/dungeonrunner/views/GameOverScreen.java:1]: Makes it a lot easier to understand.

**Pull request merged**